lib/Runtime/Base/HiResTimer.h

```diff
@@ -22,7 +22,14 @@ namespace Js
 
         double GetAdjustFactor();
     public:
-        HiResTimer(): fInit(false), dBaseTime(0), baseMsCount(0),  fHiResAvailable(true), dLastTime(0), dAdjustFactor(1), fReset(true) {}
+        HiResTimer(): fInit(false), dBaseTime(0), baseMsCount(0),  
+#ifdef _WIN32
+        // xplat-todo: do we really need this for now?
+        fHiResAvailable(true), 
+#else
+        fHiResAvailable(false), 
```

Comment:
> I dont understand- it looks like the PAL provides `QueryPerformanceCounter`, yes?

A new addition to PAL ?? IIRC this is one thing we shouldn't? This method produces right results (cross compared with i.e. node). BTW; moving HiResTimer into PlatformAgnostic at this point looks reasonable though. Updating the PR.